magento/magento2
# Cannot create full datetime product attribute 4053

### Description (*)
  -  Added time picker for product edit page (for attrobutes with `frontend_type` = `date`)


### Fixed Issues (if relevant)

1. magento/magento2#4053: Cannot create full datetime product attribute

### Manual testing scenarios (*)
1. Go to product edit page
2. Find any `date` attribute
3. Try to save date & time value
4. Make sure value is saved correctly
### Contribution checklist (*)
